frontend/src/translations/es.js

```
@@ -0,0 +1,76 @@
+  tasks: 'Tareas',
+  weekly: 'Semanal',
+  monthly: 'Mensual',
+  today: 'Hoy',
+  emailSync: 'Sincronizar Email',
+  addTask: '+ Agregar Tarea',
+  profile: 'Perfil',
+  logout: 'Cerrar Sesión',
+  changePassword: 'Cambiar Contraseña',
+  welcome: 'Bienvenido',
+  yourTasks: 'Tus Tareas',
+  noTasks: 'Sin tareas aún',
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)